packages/samples/mvp-dapp/contract/deployment.toml

```diff
@@ -1,11 +1,27 @@
-[[cells]]
-name = "sample-dapp"
-enable_type_id = true
+# [[cells]]
```

Comment:
This configuration could be removed if it's useless

Reply:
This is useful and will involve some deployment-related configuration here

Reply:
> This configuration could be removed if it's useless

This is useful and will involve some deployment-related configuration here